events/scripts/content-update.js

```diff
@@ -459,7 +459,12 @@ export async function getNonProdData(env) {
   const isPreviewMode = new URLSearchParams(window.location.search).get('previewMode')
   || window.location.hostname.includes('.hlx.')
   || window.location.hostname.includes('.aem.');
-  const resp = await fetch(`/events/default/${env === 'prod' ? '' : `${env}/`}metadata${isPreviewMode ? '-preview' : ''}.json`, {
+  
+  const url = new URL(window.location.href);
+  const pathname = url.pathname;
```

Comment:
We can merge this given the release urgency but I think pathname is directly available window.location